src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrManagerBase.cs

```diff
@@ -224,7 +224,7 @@ protected async Task CommitAsync(string commitMessage, (string Name, string Emai
                 .Append(repo.RemoteUri)
                 .Prepend(repo.Mapping.DefaultRemote)
                 .Distinct()
-                .OrderBy(GitRepoUrlParser.ParseTypeFromUri, Comparer<GitRepoType>.Create(GitRepoUrlParser.OrderByLocalPublicOther));
+                .OrderRemotesByLocalPublicOther();
```

Comment:
maybe we should combine this method with `Distinct()` feels like we're gonna use it with it every time

Reply:
I am not sure that is a good reason. It will not make it more readable because it will not be transparent this happens inside of an `Order*()` method but also it doesn't hurt anything to have `Distinct()` in a couple of places as it clearly states the intent.
So I wouldn't blindly try to chase down the LoC count down for all sakes.

Reply:
Sure, let's keep it as is, it's true that it's much more readable this way